src/c++/perf_analyzer/client_backend/triton/triton_client_backend.cc

```diff
@@ -28,8 +28,8 @@
 
 #include "json_utils.h"
 
-namespace perfanalyzer { namespace clientbackend {
-
+namespace triton { namespace perfanalyzer { namespace clientbackend {
```

Comment:
@deadeyegoodwin  Do we really need root namespace for perfanalyzer files as `triton`? This looks like a long chain. 
We can keep the root namespace as perfanalyzer and the client backend for triton can then be simply:
`namespace perfanalyzer { namespace clientbackend {  namespace triton {`

Reply:
For consistency all the Triton code has "triton" as the top-level namespace. It would be strange to have the perf_analyzer be different.